src/FINDOLOGIC/Export/CSV/CSVExporter.php

```diff
@@ -96,4 +97,19 @@ protected function getPropertyHeadingPart(): string
 
         return $propertyHeading;
     }
+
+    /**
+     * Returns the export file path
+     *
+     * @param string $targetDirectory
+     * @return string
+     */
+    protected function getTargetPath(string $targetDirectory): string
+    {
+        if (substr($targetDirectory, -4) === '.csv') {
```

Comment:
:thinking: A path ending in `.csv` does not mean that this is a valid place to write a file too. It's also possible to have directories ending in `.csv`.

Reply:
:memo: I guess it's safe not to use `mb_substr` here.

Reply:
Implementation was changed. This no longer applies.